gravitee-apim-rest-api/gravitee-apim-rest-api-test-fixtures/src/main/java/fixtures/repository/MessageLogFixtures.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright © 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package fixtures.repository;
+
+import io.gravitee.rest.api.model.v4.connector.ConnectorType;
+import io.gravitee.rest.api.model.v4.log.message.BaseMessageLog;
+import io.gravitee.rest.api.model.v4.log.message.MessageOperation;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Supplier;
+
+public class MessageLogFixtures {
+
+    private static Supplier<BaseMessageLog.BaseMessageLogBuilder<?, ?>> BASE_BUILDER_SUPPLIER = () ->
```

Comment:
Here I use a supplier because helper functions (as `aMessageLogWithMessageId(String id)` are modifying the state of `BASE` object. Since it's originally static, it means the default state is shared for every test, which can introduce flakiness. That's the point I want to discuss in a meeting (because the problem can heavily happen in  the other fixture files)